oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/DocumentStoreIndexerBase.java

```diff
@@ -311,6 +313,10 @@ public void reindex() throws CommitFailedException, IOException {
                 .build());
 
         indexerSupport.postIndexWork(copyOnWriteStore);
+        log.info("[TASK:FULL_INDEX_CREATION:START] Metrics {}", MetricsFormatter.newBuilder()
```

Comment:
Here it should be `[TASK:FULL_INDEX_CREATION:END]`, right?
Besides this, the naming convention is fine by me.

Reply:
Yes, good catch. Fixed.